man/fi_efa.7.md

```diff
@@ -23,16 +34,27 @@ The following features are supported:
 : The provider supports endpoint type *FI_EP_DGRAM*, and *FI_EP_RDM* on a new
   Scalable (unordered) Reliable Datagram protocol (SRD). SRD provides support
   for reliable datagrams and more complete error handling than typically seen
-  with other Reliable Datagram (RD) implementations. The EFA provider provides
+  with other Reliable Datagram (RD) implementations. The `efa` fabric provides
   segmentation, reassembly of out-of-order packets to provide send-after-send
-  ordering guarantees to applications via its *FI_EP_RDM* endpoint.
+  (SAS) ordering guarantees to applications via its *FI_EP_RDM* endpoint.
+  The `efa-direct` fabric doesn't provide such segmentation and reassembly.
```

Comment:
I would suggest removing this line because you talk about the exact capability differences below

Reply:
Done